mily/table_interface_examples.py

```diff
@@ -0,0 +1,69 @@
+from .table_interface import MFunctionTableInterfaceWidget
+from .widgets import MText, MComboBox, MISpin, MFSpin
+
+
+def partialclass(cls, partial_kwargs):
+    '''Returns a partial class with 'partial_kwargs' values set
+
+
+    This function returns a class whereby any args/kwargs in the dictionary
+    ``partial_kwargs`` are set.
+
+    Parameters
+    ----------
+    partial_kwargs : dict
+        a dict mapping arg/kwarg parameters to values.
+    '''
+
+    class PartialClass(cls):
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **partial_kwargs, **kwargs)
```

Comment:
may want to do


```suggestion
            super().__init__(*args, **{**partial_kwargs, **kwargs})
```

as that will make sure any overlap betweer `partial_kwargs` and `kwargs` will be resolved in `kwargs` favor, currently it will raise an complain the same kwarg was passed in twice.

Reply:
this is a good call, thanks @tacaswell. The question is do we want it to resolve in ``kwargs`` favour or ``partial_kwargs`` favour? My gut reaction is ``partial_kwargs`` as that is being defined using the PartialClass call and hence the person coding might expect it to resolve in favour of ``partial_kwargs``, but it is not a strong opinion. what do you think?

Reply:
In a group code-review at NSLS-II @tacaswell convinced me that `kwargs`` should be favoured, so I have made that change in the next commit.